From 75cd93cfa68284f512e35029ecbe640ed611e169 Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Thu, 24 Aug 2017 18:27:51 +0800
Subject: [PATCH] UPSTREAM: dt-bindings: rockchip-thermal: Support the RV1108
 SoC compatible

Add a new compatible for thermal founding on RV1108 SoCs.

Acked-by: Rob Herring <robh@kernel.org>
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
Signed-off-by: Eduardo Valentin <edubezval@gmail.com>
(cherry picked from commit 1027d759c9517a255f663d4365d98d78d8680b20)

Change-Id: I280b0dc88ff0ff00618617767b267846271fce61
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 Documentation/devicetree/bindings/thermal/rockchip-thermal.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
index bee8cf5b4710..0523e433b82d 100644
--- a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
@@ -11,6 +11,7 @@ Required properties:
    "rockchip,rk3366-tsadc": found on RK3366 SoCs
    "rockchip,rk3368-tsadc": found on RK3368 SoCs
    "rockchip,rk3399-tsadc": found on RK3399 SoCs
+   "rockchip,rv1108-tsadc": found on RV1108 SoCs
 - reg : physical base address of the controller and length of memory mapped
 	region.
 - interrupts : The interrupt number to the cpu. The interrupt specifier format
-- 
2.35.3

